webapp/channels/src/components/channel_info_rhs/header.tsx

```diff
@@ -41,23 +41,25 @@ const Header = ({channel, isArchived, isMobile, onClose}: Props) => {
                         />
                     </button>
                 )}
-                <HeaderTitle
-                    id='rhsPanelTitle'
-                >
-                    <FormattedMessage
-                        id='channel_info_rhs.header.title'
-                        defaultMessage='Info'
-                    />
-                </HeaderTitle>
+                <h2>
+                    <HeaderTitle
+                        id='rhsPanelTitle'
+                    >
+                        <FormattedMessage
+                            id='channel_info_rhs.header.title'
+                            defaultMessage='Info'
+                        />
+                    </HeaderTitle>
 
-                {channel.display_name &&
-                <span
-                    className='style--none sidebar--right__title__subtitle'
-                >
-                    {isArchived && (<Icon className='icon icon-archive-outline'/>)}
-                    {channel.display_name}
-                </span>
-                }
+                    {channel.display_name &&
```

Comment:
I don't think the channel name should be inside of the `h2` tag for this and the `ChannelMembersRhs` below. It arguably could be, but we have the channel name outside of it in the search results at least

Reply:
Ah, I missed that Matt had suggested that above.

We also discussed this earlier in our 1-1 (before I read the above comment), and agreed on including these in the `h2` elements so that it'd be read out by the screen reader